BlitzBoard/Features/LoginScreenViewController.swift

```diff
@@ -0,0 +1,48 @@
+// LoginScreenViewController.swift
+// BlitzBoard
+//
+// Created by Kiasha Rangasamy on 2024/04/11.
+//
+import UIKit
+
+class LoginScreenViewController: UIViewController, UITextFieldDelegate {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet private weak var emailTextField: UITextField!
+    @IBOutlet private weak var passwordTextField: UITextField!
+    
+    // MARK: IBActions
+    
+    @IBAction private func loginButtonTapped(_ sender: Any) {
+        let email = emailTextField.text ?? ""
+        let password = passwordTextField.text ?? ""
+        if email == "kiashar@gmail.com" && password == "Kiasha1006" {
+            print("Login successful!")
+        } else {
+            showAlert(message: "Please check login credentials.")
+        }
+    }
+    
+    // MARK: - Functions
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        emailTextField.delegate = self
+        passwordTextField.delegate = self
+    }
+    
+    // MARK: - Public Functions
+    
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
```

Comment:
what does this do?

Reply:
I removed it now, I don't think it was necessary to have